pkg/alertmanager/multitenant.go

```diff
@@ -500,6 +507,119 @@ func (am *MultitenantAlertmanager) starting(ctx context.Context) (err error) {
 	return nil
 }
 
+// migrateStateFilesToPerTenantDirectories migrates any existing configuration from old place to new hierarchy.
+// TODO: Remove in Cortex 1.10.
+func (am *MultitenantAlertmanager) migrateStateFilesToPerTenantDirectories() error {
+	migrate := func(from, to string) error {
+		level.Info(am.logger).Log("msg", "migrating AM state", "from", from, "to", to)
```

Comment:
I would be crystal clear in log messages:

```suggestion
		level.Info(am.logger).Log("msg", "migrating alertmanager state", "from", from, "to", to)
```